### DISCUSSION TITLE
Update to 1.1.6 but not recognize the previous plots

### DISCUSSION THREAD

**Discussion by LTM8888** [Chia-Network/chia-blockchain#6072] [Support]
Hi , 
I have upgrade from 1.1.5 to 1.1.6 yesterday. However, the 1.1.6 was not recognized my previous plots (approx 400+) . Please kindly advise.

I also have another problem. I have two computers farming and trying to assign one of the computer as main farmer (192.168.1.29) and the other as harvester.(192.168.1.36)

I have used the follow script to convert one the computer as harvest but did not showed up in farmer PC. Please advise if I have missing steps. Thank you.

cd "C:\Users\Chia_Farmer_PC\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon\"

.\chia configure --set-farmer-peer 192.168.1.29:8447

.\chia configure --enable-upnp false

**Comment by BrandtH22** [comment]
@LTM8888 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
